docs/flow/reference/components/prometheus.integration.node_exporter.md

```diff
@@ -0,0 +1,315 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/prometheus.integration.node_exporter
+title: prometheus.integration.node_exporter
+---
+
+# prometheus.integrations.node_exporter
+The `prometheus.integrations.node_exporter` embeds 
+[node_exporter](https://github.com/prometheus/node_exporter) which exposes a 
+wide variety of hardware and OS metrics for \*nix-based systems.
+
+The `node_exporter` itself is comprised of various _collectors_, which can be
+enabled and disabled at will. For more information on collectors, refer to the
+[`collectors-list`](#collectors-list) section.
+
+The `prometheus.integrations.node_exporter` component is a _singleton_
+component. That means that it can only appear once per configuration file, and
+a block label must not be passed to it.
```

Comment:
Do we want to say "unique component" instead of "singleton component?" cc @mattdurham

Reply:
I like that, that seems preferable over establishing a term